CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ option(ENABLE_GLES "Enable GL ES driver" OFF)
 option(ENABLE_VULKAN "Enable Vulkan driver" ON)
 option(ENABLE_RENDERDOCCMD "Enable renderdoccmd" ON)
 option(ENABLE_QRENDERDOC "Enable qrenderdoc" ON)
+option(ENABLE_VRAPI "Enable GearVR driver" OFF)
```

Comment:
This doesn't need to have a separate toggle - since it doesn't have any complex external dependencies, it can just be always compiled in. I think eventually we can remove the GLES toggle either, but there might still be some things it depends on the external system having headers (I'm not sure) so it might not always build.